optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/bavet/bi/BavetAbstractBiConstraintStream.java

```diff
@@ -111,6 +113,19 @@ protected void addChildStream(BavetAbstractBiConstraintStream<Solution_, A, B> c
         throw new UnsupportedOperationException();
     }
 
+    @Override
+    public <GroupKey_, ResultContainer_, Result_> BiConstraintStream<GroupKey_, Result_> groupBy(
+            BiFunction<A, B, GroupKey_> groupKeyMapping,
+            BiConstraintCollector<A, B, ResultContainer_, Result_> collector) {
+        throw new UnsupportedOperationException();
+    }
+
+    @Override
+    public <GroupKeyA_, GroupKeyB_> BiConstraintStream<GroupKeyA_, GroupKeyB_> groupBy(
+            BiFunction<A, B, GroupKeyA_> groupKeyAMapping, BiFunction<A, B, GroupKeyB_> groupKeyBMapping) {
+        throw new UnsupportedOperationException();
```

Comment:
If Drools supports these, don't we want to put the UnsupportedOperationException throwing in the Bavet impl?

Reply:
Not sure what you mean. It *is* in Bavet impl.

Reply:
Good point :)